pkg/policy/api/rule.go

```diff
@@ -458,3 +486,17 @@ const (
 // KafkaTopicValidChar is a one-time regex generation of all allowed characters
 // in kafka topic name.
 var KafkaTopicValidChar = regexp.MustCompile(`^[a-zA-Z0-9\\._\\-]+$`)
+
+// GetAPIKey returns the APIKey as integer
+func (kr *PortRuleKafka) GetAPIKey() int16 {
```

Comment:
Not too important. You check for the -1 case every time you use this function so you can just return a second value to indicate that it's valid.

Reply:
agreed, fixed